spec/acceptance/validate_ruby_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'spec_helper_acceptance'
 
-describe 'Running ruby validation' do
+describe 'pdk validate ruby', module_command: true do
```

Comment:
Apologies, I looked at that document and totally overlooked that section.
I still feel this method is a little bit 'magic', but if it's right there in the rspec docs then I think it's OK.